src/test/java/com/google/sps/data/EmailTest.java

```diff
@@ -0,0 +1,119 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import static org.mockito.Mockito.when;
+
+import com.google.sps.data.Email;
+import com.google.sps.data.Post;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+/** Tests creating and converting HTML for email sending. */
+@RunWith(JUnit4.class)
+public final class EmailTest {
+  
+  public static final String PATH = "path_test";
+  @Mock private static File mfile;
+  @Mock private static FileUtils mFileUtils;
+  @Mock private static Post mPost;
+
+  private static Email email;
+ 
+  @Before
+  public void setUp() {
+    
+    email = new Email();
+    MockitoAnnotations.initMocks(this);
+  }
+
+  @Test
+  public void getStringFromAvailableHTML() throws IOException {
+    // Tests if welcomeContentPath exists.
+
+    email.getWelcomeString();
```

Comment:
Assert something here e.g. that the welcome string matches the expected welcome string.